webpack-dev-server fails with following error

```
ERROR in ./~/css-loader?{"minimize":false}!./~/vue-loader/lib/style-compiler?{"vue":true,"id":"data-v-0513ca74","scoped":true,"hasInlineConfig":false}!./~/postcss-loader/lib?{"sourceMap":true}!./~/resolve-url-loader!./~/vue-loader/lib/selector.js?type=styles&index=0!./app/javascript/packs/app.vue
Module build failed: Error: Loading PostCSS Plugin failed: Cannot find module 'cssnext'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at load (/home/aks/playground/formester/node_modules/postcss-load-plugins/lib/plugins.js:44:18)
    at /home/aks/playground/formester/node_modules/postcss-load-plugins/lib/plugins.js:66:18
    at Array.forEach (native)
    at plugins (/home/aks/playground/formester/node_modules/postcss-load-plugins/lib/plugins.js:65:8)
    at /home/aks/playground/formester/node_modules/postcss-load-config/index.js:64:18
 @ ./~/vue-style-loader!./~/css-loader?{"minimize":false}!./~/vue-loader/lib/style-compiler?{"vue":true,"id":"data-v-0513ca74","scoped":true,"hasInlineConfig":false}!./~/postcss-loader/lib?{"sourceMap":true}!./~/resolve-url-loader!./~/vue-loader/lib/selector.js?type=styles&index=0!./app/javascript/packs/app.vue 4:14-434
 @ ./app/javascript/packs/app.vue
 @ multi (webpack)-dev-server/client?http://0.0.0.0:8080 ./app/javascript/packs/app.vue

ERROR in ./~/css-loader?{"minimize":false}!./~/vue-loader/lib/style-compiler?{"vue":true,"id":"data-v-79783b26","scoped":false,"hasInlineConfig":false}!./~/postcss-loader/lib?{"sourceMap":true}!./~/resolve-url-loader!./~/vue-loader/lib/selector.js?type=styles&index=0!./app/javascript/components/flash.vue
Module build failed: Error: Loading PostCSS Plugin failed: Cannot find module 'cssnext'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at load (/home/aks/playground/formester/node_modules/postcss-load-plugins/lib/plugins.js:44:18)
    at /home/aks/playground/formester/node_modules/postcss-load-plugins/lib/plugins.js:66:18
    at Array.forEach (native)
    at plugins (/home/aks/playground/formester/node_modules/postcss-load-plugins/lib/plugins.js:65:8)
    at /home/aks/playground/formester/node_modules/postcss-load-config/index.js:64:18
 @ ./~/vue-style-loader!./~/css-loader?{"minimize":false}!./~/vue-loader/lib/style-compiler?{"vue":true,"id":"data-v-79783b26","scoped":false,"hasInlineConfig":false}!./~/postcss-loader/lib?{"sourceMap":true}!./~/resolve-url-loader!./~/vue-loader/lib/selector.js?type=styles&index=0!./app/javascript/components/flash.vue 4:14-437
 @ ./app/javascript/components/flash.vue
 @ ./app/javascript/packs/shared/flash.js
 @ multi (webpack)-dev-server/client?http://0.0.0.0:8080 ./app/javascript/packs/shared/flash.js

ERROR in ./~/css-loader?{"minimize":false}!./~/vue-loader/lib/style-compiler?{"vue":true,"id":"data-v-519b67bb","scoped":false,"hasInlineConfig":false}!./~/postcss-loader/lib?{"sourceMap":true}!./~/resolve-url-loader!./~/vue-loader/lib/selector.js?type=styles&index=0!./app/javascript/components/button_dropdown.vue
Module build failed: Error: Loading PostCSS Plugin failed: Cannot find module 'cssnext'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at load (/home/aks/playground/formester/node_modules/postcss-load-plugins/lib/plugins.js:44:18)
    at /home/aks/playground/formester/node_modules/postcss-load-plugins/lib/plugins.js:66:18
    at Array.forEach (native)
    at plugins (/home/aks/playground/formester/node_modules/postcss-load-plugins/lib/plugins.js:65:8)
    at /home/aks/playground/formester/node_modules/postcss-load-config/index.js:64:18
 @ ./~/vue-style-loader!./~/css-loader?{"minimize":false}!./~/vue-loader/lib/style-compiler?{"vue":true,"id":"data-v-519b67bb","scoped":false,"hasInlineConfig":false}!./~/postcss-loader/lib?{"sourceMap":true}!./~/resolve-url-loader!./~/vue-loader/lib/selector.js?type=styles&index=0!./app/javascript/components/button_dropdown.vue 4:14-447
 @ ./app/javascript/components/button_dropdown.vue
 @ ./app/javascript/packs/shared/hero.js
 @ multi (webpack)-dev-server/client?http://0.0.0.0:8080 ./app/javascript/packs/shared/hero.js
webpack: Failed to compile.
```

--- a/README.md
+++ b/README.md
@@ -366,7 +366,7 @@ file in your app root with standard plugins.
 ```yml
 plugins:
   postcss-smart-import: {}
-  cssnext: {}
+  postcss-cssnext: {}
 ```
 
 
@@ -897,7 +897,7 @@ const Hello = props => (
 ### CSS-Next
 
 [css-next](http://cssnext.io/) is supported out-of-box in Webpacker allowing the use of
-latest css features, today.
+latest CSS features, today.
 
 
 ### Ignoring swap files

--- a/lib/install/config/.postcssrc.yml
+++ b/lib/install/config/.postcssrc.yml
@@ -1,3 +1,3 @@
 plugins:
   postcss-smart-import: {}
-  cssnext: {}
+  postcss-cssnext: {}

